pkg/k8sutil/k8sutil.go

```diff
@@ -221,7 +221,7 @@ func (k *K8sutil) CreateKubernetesCustomResourceDefinition() error {
 		logrus.Infof("SKIPPING: already exists %#v\n", crd.ObjectMeta.Name)
 	}
 
-	return nil
+	return crd, err
```

Comment:
This should be: `return crd, nil` otherwise on first install, if the CRD doesn't exist, it will return the value of `err` from the first test. This breaks the controller `init()` function.